observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,61 @@ These release notes track the development of the Network Observability Operator
 
 For an overview of the Network Observability Operator, see xref:../../observability/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
 
+[id="network-observability-operator-release-notes-1-6"]
+== Network Observability Operator 1.6.0
+The following advisory is available for the Network Observability Operator 1.6.0:
+// note to self: update errata live ID
+* link:https://access.redhat.com/errata/RHSA-2024:0853[Network Observability Operator 1.6.0]
+
+[id="network-observability-operator-1.6.0-features-enhancements"]
+=== New features and enhancements
+
+[id="network-observability-custom-metrics-1.6"]
+==== Custom metrics API
+You can create custom metrics out of the flowlogs data using the `FlowMetrics` API. Flowlogs data can be used with Prometheus labels to customize cluster information on your dashboards. For more information, see xref:../../observability/network_observability/metrics-alerts-dashboards.adoc#network-observability-custom-metrics_metrics-dashboards-alerts[Custom metrics] and <FLOW_METRICS_API_REF_LINK_HERE>
+
+[id="network-observability-eBPF-performance-enhancements-1.6"]
+==== eBPF performance enhancements
+*NetObserv / Health* dashboard integration with eBPF metrics allows you to view alerts in two causes: when the Network Observability eBPF agent hashmap table is full, which cases the eBPF agent processes flows with degraded performance, or when the capacity limiter is triggered. For more information, see xref:../../observability/network_observability/network-observability-operator-monitoring.adoc#network-observability-netobserv-dashboard-ebpf-agent-alerts_network_observability[Using the eBPF agent alert]. 
+
+[id="network-observability-cli-1.6"]
+==== Network Observability CLI
```

Comment:
Mention that it is tech preview?

Reply:
OH yes, I need to create a TP section in our release notes